src/ngx_http_lua_headers.c

```diff
@@ -145,10 +160,36 @@ ngx_http_lua_ngx_req_raw_header(lua_State *L)
 
     dd("size: %d", (int) size);
 
+#if defined(nginx_version) && nginx_version >= 1011011
+    if (hc->nbusy && ctx->busy_bufs_ptrs == NULL) {
+        ctx->busy_bufs_ptrs = ngx_array_create(r->pool, hc->nbusy, sizeof(ngx_buf_t *));
```

Comment:
I still don't see why we should use `ngx_array_t` here, as mentioned in my previous round of review. This array never needs to grow IMHO. It's more efficient both CPU-wise and memory-wise to just allocate a flat `ngx_buf_t` C array here.